src/pages/AppDetail.svelte

```diff
@@ -33,6 +34,13 @@
   }
 </style>
 
+{#if app}
+  {#if app.prototype}
+    <AppBanner
+      message="The glean-js application is in prototype stage. The metrics listed below may contain inconsistencies and testing string."
```

Comment:
Since the app page now lists the pings as well as the metrics, let's tweak this message just a bit so that it makes more sense, let me know what you think of:

```suggestion
      message="The glean-js application is a prototype. The metrics and pings listed below may contain inconsistencies and testing strings."
```

Reply:
The new message works better. I'll change it to that.